### PR TITLE
Create basic layout

### PR DESCRIPTION
**Pull request by benedihm**

### CODE REVIEW

File: backend/src/routes/ontologies.ts [hunk 1]
```diff
@@ -24,6 +26,16 @@ const getSubclassesFromClass = async (req, res) => {
   }
 };
 
+const getannotationsFromClass = async (req, res) => {
+  try {
+    const data = await getannotations(req.params.classId);
+    res.json(data);
+  } catch (e) {
+    const status = e instanceof ApiError ? e.statusCode : 500;
```
Comment (AndreasWintherMoen) [inline backend/src/routes/ontologies.ts:34]: Use the onError middleware instead. See the other routes.

File: backend/src/database/queries/getAnnotations.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    ${prefixString}
+    PREFIX rdfs1: <http://www.semanticweb.org/aga/ontologies/2017/9/SDG#rdfs:>
+    PREFIX : <http://www.semanticweb.org/aga/ontologies/2017/9/SDG#:>
+    PREFIX schema: <http://schema.org/>
+    select *
+    where { 
+    { 
+         ${fullClassName} rdfs:label ?label.
+         ${fullClassName} rdfs1:description ?description.
+         SDG:Miljø rdfs:label ?relation
```
Comment (erikskaar) [inline backend/src/database/queries/getAnnotations.ts:25]: Not sure if we should implement a hardcoded query to check that the query works (enable buttons in the frontend). I think it should be implemented in another way that doesn't require hardcoding.

**Review by erikskaar — CHANGES_REQUESTED** [review]

File: backend/src/database/getRelations.ts [hunk 3]
```diff
@@ -27,7 +26,7 @@ export default async (classId: string): Promise<Array<Ontology>> => {
   const ontologies = response.records
     .map(mapRecordToOntology)
     .map((ont) => addEntityToNullFields(ont, ontologyEntity))
-    .filter(isRelevantOntology)
-    .filter(isNotLoopOntology);
+    .filter(isRelevantOntology);
+  // .filter(isNotLoopOntology);
```
[the remark this answers is not in the document]
Comment (Kapteinkrok) [reply backend/src/database/getRelations.ts:30]: This has been left as is: For some reason the "isNotLoopOntology" and its correlating .filter function gives errors.  By comparing it with Main, and having a identical code, I see no reason why it should not work, however it does not^^

[the remark this answers is not in the document]
Comment (erikskaar) [reply backend/src/database/getRelations.ts:30]: That does imply that there has been made a mistake somewhere that breaks it. I will take a look and see if I find anything :)

[the remark this answers is not in the document]
Comment (erikskaar) [reply backend/src/database/getRelations.ts:30]: It looks like you have forgotten to remove the semicolon from the line above when uncommenting. Only the last line needs a semicolon as they are all technically on the same line. I will push this change.

**Review by erikskaar — APPROVED** [review]
Should be fine now 🍾

**Review by AndreasWintherMoen — APPROVED** [review]
LGTM

**Pull request merged**